transcom/mymove
# INT B-18631 All shipment weight values displayed sc closeout

## [B-18631](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18631)

## Summary

This adds the ability to see all HHG shipment weights while Closeout SC is closing out a member's PPM. 
This is to be displayed to the closeout counselor during their review of the PPM weights.

### How to test
Test using `ApprovedMoveWithPPMWeightTicketOfficeWithHHG` or create your own move.

With `ApprovedMoveWithPPMWeightTicketOfficeWithHHG`:
1. Use `ApprovedMoveWithPPMWeightTicketOfficeWithHHG` testharness.
2. Find the move as Closeout SC -> review the documents -> You should see `Estimated Weight` and `Approved Weight` above `PPM 1` on the right side bar.

With your own move:
1. Create a move with at least one HHG and a PPM shipment.
2. As a SC -> Submit the move.
3. As a TOO -> Approve HHG.
4. As a Prime -> `Update Shipment` in Prime Sim for the HHG move.
5. Fill in the appropriate fields and make note of your `Actual Weight` and `Estimated Weight`.
6. As a customer -> Upload your PPM documents.
7. As a closeout SC -> use the search function to find the move.
8. Review the PPM document and you should see `Estimated Weight` and `Approved Weight` above `PPM 1` on the right side bar.

## Screenshots
![image](https://github.com/transcom/mymove/assets/146856854/0d67a6eb-53e4-4fa6-8ecc-db691a74d444)
